### PR TITLE
First entry in wiki history leads to newest revision.

### PR DESCRIPTION
**Pull request by yglukhov**
Getting to newest revision of a wiki page from its history lead to a page with a notice, that it's not newest. This PR fixes this unpleasantness.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://yglukhov-wiki-history.c68bfdc4cac4c32c41e1a922e73062ea.ttrcloud.com' target='_blank'>Click to open.</a>

File: app/views/projects/wikis/history.html.haml [hunk 1]
```diff
@@ -12,11 +12,12 @@
...
       %th Format
   %tbody
-    - @page.versions.each do |version|
+    - @page.versions.each_with_index do |version, index|
       - commit = version
       %tr
         %td
-          = link_to project_wiki_path(@project, @page, version_id: commit.id) do
+          = link_to project_wiki_path(@project, @page,
+              index == 0 ? {} : {version_id: commit.id}) do
```
Comment (randx) [inline app/views/projects/wikis/history.html.haml]: can you refactor this? Maybe move `index == 0 ? {} : {version_id: commit.id}` in separate variable

**Comment by yglukhov** [comment]
@randx, do you mean adding a helper function or just do it inline in the template?

**Comment by randx** [comment]
@yglukhov helper function is more prefferable

**Comment by yglukhov** [comment]
Added a helper func.

**Comment by randx** [comment]
thank you

**Comment by yglukhov** [comment]
Thanx!

**Pull request merged**